### PR TITLE
Fix navigation menu

### PR DESCRIPTION
**Pull request by Arhell**
Hello!
See issue #19899
This PR fix

### CODE REVIEW

**Comment by Arhell** [comment]
Before --> https://kubernetes.io/blog/
after --> https://deploy-preview-19926--kubernetes-io-master-staging.netlify.com/blog/

File: static/css/blog.css [hunk 1]
```diff
@@ -497,6 +497,10 @@ img.big-img {
   overflow-x: scroll;
 }
 
+.global-nav {
+  padding: 0;
```
Comment (kbhawkey) [inline static/css/blog.css]: Hello @Arhell ,
I looked at the preview and compared the `ul.global-nav` setting for the Documentation versus the Blog.
The change makes the Blog top nav match the other pages from the top nav.
From a quick inspection, the Documentation item picks up the padding from `ul`.
I'd like someone from the Blog sub-project team to look at the change.
LGTM
/cc @kbarnard10

**Comment by jimangel** [comment]
@Arhell please address conflicting files. Thanks!

**Comment by Arhell** [comment]
> @Arhell please address conflicting files. Thanks!

done

**Pull request merged**